boltdesignsystem/bolt
# DS-545 add pin icon and fix name typo

## Jira

https://pegadigitalit.atlassian.net/browse/DS-545

## Summary

Adds a pin icon, fixes a typo in the `pega-connection-hierarchy` icon name, and fixes the stroke width in `pega-database` icon.

## Details

1. Added Pin icon
2. Fixed `pega-connection-hierarchy` typo
3. Fixed database icon stroke width

## How to test

Run the branch locally and check the icon docs. Make sure all 3 icons display correctly.
